2018-08-30 20:47:36,707- scoreDao.py:[line:59]- INFO - 用户or6FL0xFVD-QR-2Taj9b7xDyESIc数据库中无成绩
2018-08-30 20:47:36,709- userDao.py:[line:31]- INFO - 用户名 or6FL0xFVD-QR-2Taj9b7xDyESIc 以绑定
2018-08-30 20:48:07,022- scoreDao.py:[line:128]- ERROR - 用户or6FL0xFVD-QR-2Taj9b7xDyESIc成绩查询失败
2018-08-30 20:48:07,022- scoreDao.py:[line:66]- WARNING - 用户or6FL0xFVD-QR-2Taj9b7xDyESIc已爬1次
2018-08-30 20:48:07,025- scoreDao.py:[line:59]- INFO - 用户or6FL0xFVD-QR-2Taj9b7xDyESIc数据库中无成绩
2018-08-30 20:48:07,026- userDao.py:[line:31]- INFO - 用户名 or6FL0xFVD-QR-2Taj9b7xDyESIc 以绑定
2018-08-30 20:48:35,082- userDao.py:[line:31]- INFO - 用户名 or6FL0xFVD-QR-2Taj9b7xDyESIc 以绑定
2018-08-30 20:48:37,318- scoreDao.py:[line:128]- ERROR - 用户or6FL0xFVD-QR-2Taj9b7xDyESIc成绩查询失败
2018-08-30 20:48:37,319- scoreDao.py:[line:66]- WARNING - 用户or6FL0xFVD-QR-2Taj9b7xDyESIc已爬2次
2018-08-30 20:49:53,390- scoreDao.py:[line:59]- INFO - 用户or6FL0xFVD-QR-2Taj9b7xDyESIc数据库中无成绩
2018-08-30 20:49:53,392- userDao.py:[line:31]- INFO - 用户名 or6FL0xFVD-QR-2Taj9b7xDyESIc 以绑定
2018-08-30 20:50:23,462- scoreDao.py:[line:128]- ERROR - 用户or6FL0xFVD-QR-2Taj9b7xDyESIc成绩查询失败
2018-08-30 20:50:23,463- scoreDao.py:[line:66]- WARNING - 用户or6FL0xFVD-QR-2Taj9b7xDyESIc已爬1次
2018-08-30 20:50:23,465- scoreDao.py:[line:59]- INFO - 用户or6FL0xFVD-QR-2Taj9b7xDyESIc数据库中无成绩
2018-08-30 20:50:23,467- userDao.py:[line:31]- INFO - 用户名 or6FL0xFVD-QR-2Taj9b7xDyESIc 以绑定
2018-08-30 20:50:53,535- scoreDao.py:[line:128]- ERROR - 用户or6FL0xFVD-QR-2Taj9b7xDyESIc成绩查询失败
2018-08-30 20:50:53,536- scoreDao.py:[line:66]- WARNING - 用户or6FL0xFVD-QR-2Taj9b7xDyESIc已爬2次
2018-08-30 21:29:49,553- userDao.py:[line:31]- INFO - 用户名 or6FL0xFVD-QR-2Taj9b7xDyESIc 以绑定
2018-08-30 21:29:49,554- userDao.py:[line:31]- INFO - 用户名 or6FL0xFVD-QR-2Taj9b7xDyESIc 以绑定
2018-08-30 21:29:54,706- userDao.py:[line:31]- INFO - 用户名 or6FL0xFVD-QR-2Taj9b7xDyESIc 以绑定
2018-08-30 21:29:54,711- userDao.py:[line:73]- INFO - 用户or6FL0xFVD-QR-2Taj9b7xDyESIc解绑成功
2018-08-30 21:30:03,884- userDao.py:[line:28]- INFO - 用户名 or6FL0xFVD-QR-2Taj9b7xDyESIc 以解绑
2018-08-30 21:30:07,765- userDao.py:[line:28]- INFO - 用户名 or6FL0xFVD-QR-2Taj9b7xDyESIc 以解绑
2018-08-30 21:30:14,374- userDao.py:[line:31]- INFO - 用户名 or6FL0xFVD-QR-2Taj9b7xDyESIc 以绑定
2018-08-30 21:30:14,374- userDao.py:[line:55]- ERROR - 用户or6FL0xFVD-QR-2Taj9b7xDyESIc用户绑定失败
2018-08-30 21:30:18,835- userDao.py:[line:31]- INFO - 用户名 or6FL0xFVD-QR-2Taj9b7xDyESIc 以绑定
2018-08-30 21:30:18,835- userDao.py:[line:55]- ERROR - 用户or6FL0xFVD-QR-2Taj9b7xDyESIc用户绑定失败
2018-08-30 21:30:25,533- userDao.py:[line:31]- INFO - 用户名 or6FL0xFVD-QR-2Taj9b7xDyESIc 以绑定
2018-08-30 21:30:25,534- userDao.py:[line:55]- ERROR - 用户or6FL0xFVD-QR-2Taj9b7xDyESIc用户绑定失败
2018-08-30 21:30:29,283- userDao.py:[line:28]- INFO - 用户名 or6FL0xFVD-QR-2Taj9b7xDyESIc 以解绑
2018-08-30 22:23:23,109- userDao.py:[line:25]- INFO - 用户名 or6FL07J8AeXcejGwN9HIUTvKiQc 为空
2018-08-30 22:23:33,979- userDao.py:[line:25]- INFO - 用户名 or6FL07J8AeXcejGwN9HIUTvKiQc 为空
2018-08-30 22:23:33,985- userDao.py:[line:58]- INFO - 用户or6FL07J8AeXcejGwN9HIUTvKiQc用户绑定成功
